modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -13,6 +13,38 @@ const errors = require('../../errors');
 const debug = require('debug')('bitgo:v2:utxo');
 import { Codes, VirtualSizes } from '@bitgo/unspents';
 
+export interface Output {
+  address: string;
+  amount: string;
+}
+
+export interface TransactionFee {
+  fee: number;
+  feeRate?: number;
+  size: number
+}
+
+export interface TransactionExplanation {
```

Comment:
Is this the same interface across coins? Maybe it could be moved to baseCoin

Reply:
It's not the same, but I created one in baseCoin that's extended by each coin that implements explainTransaction.